src/utils/utils.service.js

```diff
@@ -59,6 +59,47 @@ class UtilsService {
       return index
     }, {})
   }
+
+  checkAndResolveConflicts (changedDoc, pouchdb) {
+    let preferredRevision = {}
+    if (!changedDoc._conflicts) {
+      return
+    }
+
+    if (!changedDoc.updatedAt) {
+      preferredRevision = changedDoc
+      changedDoc._conflicts.map((conflictingRev) => {
+        if (conflictingRev.updatedAt) {
```

Comment:
the `if` checks that the `changedDoc` does not have `updatedAt` property so at this point `preferredRevision` does not have an `updatedAt` property so cannot be newer